src/NuGet.Services.Configuration/Properties/AssemblyInfo.cs

```diff
@@ -7,6 +7,6 @@
 [assembly: AssemblyCompany(".NET Foundation")]
 [assembly: AssemblyCopyright("Copyright © .NET Foundation 2016")]
 [assembly: ComVisible(false)]
-[assembly: AssemblyVersion("2.0.0")]
-[assembly: AssemblyFileVersion("2.0.0")]
-[assembly: AssemblyInformationalVersion("2.0.0-rc2-1")]
```

Comment:
I was changing this because this assembly isn't used yet, and these version numbers were copied from logging. If there are consequences to this, i will change it back.

---
In reply to: [74493162](https://github.com/NuGet/NuGet.Staging/pull/71#discussion_r74493162) [](ancestors = 74493162)